test/cea/cea_decoder_unit.js

```diff
@@ -25,469 +25,551 @@ describe('CeaDecoder', () => {
   /** @type {!shaka.cea.CeaDecoder} */
   const decoder = new shaka.cea.CeaDecoder();
 
-  const edmCodeByte2 = 0x2c; // Erase displayed memory byte 2.
+  describe('decodes CEA-608', () => {
+    const edmCodeByte2 = 0x2c; // Erase displayed memory byte 2.
 
-  // Blank padding control code between two control codes that are the same.
-  const blankPaddingControlCode = new Uint8Array([0x97, 0x23]);
+    // Blank padding control code between two control codes that are the same.
+    const blankPaddingControlCode = new Uint8Array([0x97, 0x23]);
 
-  // Erases displayed memory on every captioning mode.
-  const eraseDisplayedMemory = new Uint8Array([
-    ...atscCaptionInitBytes, 0xc4, /* padding= */ 0xff,
-    0xfc, 0x94, edmCodeByte2, // EDM on CC1
-    0xfc, 0x1c, edmCodeByte2, // EDM on CC2
-    0xfd, 0x15, edmCodeByte2, // EDM on CC3
-    0xfd, 0x9d, edmCodeByte2, // EDM on CC4
-  ]);
-
-  beforeEach(() => {
-    decoder.clear();
-  });
-
-  it('green and underlined popon caption data on CC3', () => {
-    const controlCount = 0x08;
-    const captionData = 0xc0 | controlCount;
-    const greenTextCC3Packet = new Uint8Array([
-      ...atscCaptionInitBytes, captionData, /* padding= */ 0xff,
-      0xfd, 0x15, 0x20, // Pop-on mode (RCL control code)
-      0xfd, 0x13, 0xe3, // PAC to underline and color text green on last row.
-      0xfd, 0x67, 0xf2, // g, r
-      0xfd, 0xe5, 0xe5, // e, e
-      0xfd, 0x6e, 0x20, // n, space
-      0xfd, 0xf4, 0xe5, // t, e
-      0xfd, 0xf8, 0xf4, // x, t
-      0xfd, 0x15, 0x2f, // EOC
```

Comment:
Nothing has actually changed in the CEA-608 tests. Github is just showing this as a diff since I tabbed all of them and put them under `describe('decodes CEA-608)`.